hpke.go

```diff
@@ -259,30 +355,30 @@ func SetupBaseR(suite CipherSuite, skR KEMPrivateKey, enc, info []byte) (*Decryp
 	}
 
 	pkR := skR.PublicKey()
-	key, nonce, err := keySchedule(suite, modeBase, pkR, zz, enc, info, defaultPSK(suite), defaultPSKID(suite), defaultPKIm(suite))
+	decryptionContextParams, err := keySchedule(suite, ModeBase, pkR, zz, enc, info, defaultPSK(suite), defaultPSKID(suite), defaultPKIm(suite))
 	if err != nil {
 		return nil, err
 	}
 
-	return newDecContext(suite, key, nonce)
+	return newDecContext(suite, SetupParameters{}, decryptionContextParams)
```

Comment:
Don't you want to be passing SetupParams here?  Also below.

Reply:
Yep, just didn't get there yet!